application/restore.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+for dbName in ${DB_NAMES}; do
+    logger -p user.info "restoring ${dbName}..."
+
+    runny s3cmd get -f ${S3_BUCKET}/${dbName}.sql.gz /tmp/${dbName}.sql.gz
+    runny gunzip -f /tmp/${dbName}.sql.gz
+
+    start=$(date +%s)
+    runny $(mysql -h ${MYSQL_HOST} -u ${MYSQL_USER} -p"${MYSQL_PASSWORD}" ${dbName} < /tmp/${dbName}.sql)
```

Comment:
You might be able to combine the gunzip command with this one ...

runny $(zcat /tmp/${dbName}.sql | mysql -h ${MYSQL_HOST} -u ${MYSQL_USER} -p"${MYSQL_PASSWORD}" ${dbName})

Reply:
if you don't mind, I think I like the separate steps for the sake of logging in `runny`